boards/arm/stm32f0l0g0/stm32g071b-disco/src/stm32g071b-disco.h

```diff
@@ -57,6 +57,17 @@
 #define GPIO_JOY_UP      (GPIO_INPUT | GPIO_PULLDOWN | GPIO_EXTI | \
                           GPIO_PORTC | GPIO_PIN4)
 
+/* OLED definitions *********************************************************/
+
+#define GPIO_SSD1306_CS    (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_HIGH| \
+                            GPIO_OUTPUT_SET|GPIO_PORTA|GPIO_PIN3)
+
+#define GPIO_SSD1306_CMD   (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_HIGH|  \
+                            GPIO_OUTPUT_SET|GPIO_PORTA|GPIO_PIN7)
+
+#define GPIO_SSD1306_RST   (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_HIGH|  \
+                            GPIO_OUTPUT_SET|GPIO_PORTA|GPIO_PIN6)
```

Comment:
Good to wrap `|` with spaces to follow similar style as in other places

Reply:
fixed